ceresdb-protocol/src/main/java/io/ceresdb/models/Value.java

```diff
@@ -72,7 +72,7 @@ public DataType getDataType() {
         return type;
     }
 
-    public Object getValue() {
+    public Object getAnyValue() {
```

Comment:
I think `getValue()` is better than `getAnyValue()`, what does any mean?

Reply:
because row.getColumn("test").getValue().getValue() is confusing

Reply:
rename to getObject()